src/library/librarycontrol.cpp

```diff
@@ -325,12 +347,32 @@ void LibraryControl::slotSelectPrevSidebarItem(double v) {
     }
 }
 
+void LibraryControl::slotToggleFocusWidget(double v) {
+    if (v <= 0 || m_pSidebarWidget == NULL) {
+        return;
+    }
+    QApplication::postEvent(m_pSidebarWidget, new QKeyEvent(
+        QEvent::KeyPress, (int)Qt::Key_Tab, Qt::NoModifier, QString(), true));
+}
+
 void LibraryControl::slotToggleSelectedSidebarItem(double v) {
     if (m_pSidebarWidget != NULL && v > 0) {
         m_pSidebarWidget->toggleSelectedItem();
     }
 }
 
+void LibraryControl::slotChooseItem(double v) {
+    if (m_pLibraryWidget == NULL) {
```

Comment:
Use nullptr instead of NULL. NULL should be eliminated from the existing code base step by step.